pkg/volume/csi/csi_attacher_test.go

```diff
@@ -1630,34 +1613,6 @@ func TestAttacherUnmountDevice(t *testing.T) {
 	}
 }
 
-// notifyWatchCreated returns a watch reaction func that closes the supplied channel once a watch is created.
-// This is useful for ensuring a watch has been created prior to injecting events.
-func notifyWatchCreated(watchCreated chan core.Action, watcher *watch.RaceFreeFakeWatcher, targetInvocation int) core.WatchReactionFunc {
```

Comment:
Since this was introduced earlier and now is removed, I'll see about rebasing this to remove it entirely.